Timing-Functions/ParseDate.js

```diff
@@ -0,0 +1,27 @@
+import { getMonthDays } from './GetMonthDays'
+
+function checkDate(date) {
+  if (date.day < 1 || date.day > getMonthDays(date.month, date.year)) {
+    throw new Error('Indalid day value.')
```

Comment:
Typo (indalid)

Reply:
Fixed in 8c7780b49ca4fba83040d4fdf389e41e5d4fbefe.